10-task/dubbo-samples-extensibility/dubbo-samples-extensibility-protocol-consumer/src/main/resources/application.yml

```diff
@@ -13,15 +13,17 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-# Specify the application name of Dubbo
-dubbo.application.name=extensibility-protocol-provider
-
-# Enable token verification for each invocation
-dubbo.provider.token=true
-
-# Specify the registry address
-dubbo.registry.address=nacos://localhost:8848?username=nacos&password=nacos
-# dubbo.registry.address=nacos://${nacos.address:localhost}:8848?username=nacos&password=nacos
-
-# Specify the port of eDubbo protocol
-dubbo.provider.protocol=edubbo
```

Comment:
Please let me know why this config is missing. This configuration is a sample of core content. Please check all the changes.

Reply:
Sorry, this is a mistake in my work, I will fix it and check if there are any similar errors.

Reply:
My working methods are all the same：
1. Copy the `application.properties` copy
2. Use the `Convert YAML and Properties File` plugin to convert the `application.properties` into `application.yml`
3. Move the annotations in the copy to `application.yml`
5. delete the copy.